lib/manageiq/appliance_console/message_configuration.rb

```diff
@@ -117,6 +121,7 @@ def configure_messaging_yaml
         end
 
         File.write(messaging_yaml_path, messaging_yaml.to_yaml)
+        File.chown(manageiq_uid, manageiq_gid, messaging_yaml_path)
```

Comment:
Should this follow [the same pattern as above](https://github.com/ManageIQ/manageiq-appliance_console/pull/165/files#diff-58f7d485041385eb768bd8d25d7e83a11ccf1a1bbf2bc4fb136fcdcf9cd6b590R282)?

Reply:
:+1: done